samples/logs.js

```diff
@@ -152,6 +152,37 @@ async function listLogEntriesAdvanced(filter, pageSize, orderBy) {
   // [END logging_list_log_entries_advanced]
 }
 
+async function tailLogEntries(logName) {
```

Comment:
non-blocking suggestion, if you pull this into its own file, it will have its own listing in `README.md` and `samples/README.md` which can be nice.

Reply:
This is a great suggestion, thanks!

Reply:
Actually, I think all of the samples need to be reorganized as a part of [this ticket](https://github.com/googleapis/nodejs-logging/issues/1003). I'll get the samples refactoring in in a separate PR if thats ok